custom_components/dahua/__init__.py

```diff
@@ -184,7 +184,7 @@ async def _async_update_data(self):
                     data.update(response)
 
                 device_type = data.get("deviceType", None)
-                if device_type == "IP Camera" or device_type is None:
+                if device_type in ["IP Camera", "31"] or device_type is None:
```

Comment:
Do you mind leaving a comment about "31". That will help for future maintenance.